src/lib/services/feature-toggle-service.ts

```diff
@@ -548,15 +548,22 @@ class FeatureToggleService {
      * GET /api/admin/projects/:project/features/:featureName
      * @param featureName
      * @param archived - return archived or non archived toggles
+     * @param projectId - provide if you're requesting the feature in the context of a specific project.
      */
     async getFeature(
         featureName: string,
         archived: boolean = false,
+        projectId?: string,
     ): Promise<FeatureToggleWithEnvironment> {
-        return this.featureStrategiesStore.getFeatureToggleWithEnvs(
-            featureName,
-            archived,
-        );
+        const feature =
+            await this.featureStrategiesStore.getFeatureToggleWithEnvs(
```

Comment:
I would rather just pass projectId down to the store as well and have that deal with not finding a toggle with the provided feature name and projectId.

Reply:
Fair enough! I didn't think of that. Could certainly have a look if you think it's worth diving into?